go.mod

```diff
@@ -137,7 +137,7 @@ require (
 	github.com/siderolabs/crypto v0.6.3
 	github.com/siderolabs/discovery-api v0.1.6
 	github.com/siderolabs/discovery-client v0.1.12
-	github.com/siderolabs/gen v0.8.4
+	github.com/siderolabs/gen v0.8.5-0.20250721103949-044d921685bb
```

Comment:
Do we need to release the `github.com/siderolabs/gen` before merging this PR?

Reply:
discussed in private - we need a tag, as we're very close to beta